src/main/resources/swagger.api/parameters/consortium-locations-limit-param.yaml

```diff
@@ -4,5 +4,5 @@ description: Limit the number of elements returned in the response.
 schema:
   type: integer
   minimum: 0
-  maximum: 1000
+  maximum: 10000
```

Comment:
keep in mind that this API uses pagination search and opensearch/elasticsearch puts a limit of 10.000 search results for ```from + size``` (or ```offset + limit```)
https://www.elastic.co/guide/en/elasticsearch/reference/current/paginate-search-results.html
https://opensearch.org/docs/latest/search-plugins/searching-data/paginate/

now, if API is called with max limit then offset must be 0, or if the use of using this API with ```offset + limit > 10.000``` is possible or quite frequent then search query needs to be changed

Reply:
we will use this without limit and offset param, fetching only 1000 can not be enought, but 10k is good. So 10000 as ES suggests as offset + limit

Reply:
I suggest to add validation in code to check that offset + limit is less then 10000

Reply:
added AC regarding this into newly created ticket: https://folio-org.atlassian.net/browse/MSEARCH-812

Reply:
Added validation for this case